### DISCUSSION TITLE
Windows CreateWebview: Invalid window handle

### DISCUSSION THREAD

**Discussion by psavery** [tauri-apps/tauri#6008] [Q&A]
I have an example that is running on both Mac and Linux just fine. However, on Windows, I am receiving the following error:

```rust
thread 'main' panicked at 'error while running application: Runtime(CreateWebview(WebView2Error(WindowsError(Error { code:
 0x80070578, message: Invalid window handle. }))))', src\main.rs:40:6
```

We are not rust experts. Does anyone know what the problem might be? Our code is [here](https://github.com/Kitware/trame-tauri/blob/master/examples/simple-cone/src-tauri/src/main.rs). I guess line 40 is just catching the exception.

**Comment by lucacastelnuovo** [comment]
@psavery  I'm also experiencing this issue. Were you able to resolve it?

**Reply by lucacastelnuovo** [reply]
This issue occurred when connecting to a Windows machine using VSCode SSH.
When I ran `npm run tauri dev` this would fail with the above mentioned error.

When running `npm run tauri dev` directly on the machine (in our case using screensharing software) everything worked as expected.

**Comment by jjh42** [comment]
I encountered this issue.
